common/src/fakeData/fakeSkillFamily.ts

```diff
@@ -0,0 +1,31 @@
+import faker from "faker";
+import { SkillCategory, SkillFamily, Skill } from "../api/generated";
+
+const generateSkillFamily = (name: string) => {
+  return {
+    category: faker.random.arrayElement([
+      SkillCategory.Behavioural,
+      SkillCategory.Technical,
+    ]),
+    description: {
+      en: faker.lorem.sentences(),
+      fr: faker.lorem.sentences(),
+    },
+    id: faker.datatype.uuid(),
+    key: name,
+    name: {
+      en: name,
+      fr: name,
+    },
+    skills: new Array<Skill>(faker.datatype.number(20)),
```

Comment:
Is this an empty array?

Reply:
Yeah, it's an array of nulls because it hadn't mattered to me at the time what each skill looked like. Thinking about it I'm realizing I should probably generate fake data for the skills as well.

Reply:
I noticed when reviewing in more depth that you were using it for the skill counts, which makes sense. But for as reusable fake data, this should probably actually populate the skill arrays by calling fakeSkill